src/time/time.go

```diff
@@ -0,0 +1,17 @@
+package time
+
+import (
+	"log"
+	"time"
+)
+
+// GetElapsedTime can be used to measure the functions elapsed time.
+// Use defer GetElapsedTime("functionName")() at the beginning of the functions
+// Args:
+//  	what (string): Name of the function to calculate the elapsed time.
+func GetElapsedTime(what string) func() {
```

Comment:
what do you think rename this function to PrintElapsedTIme @esequielvirtuoso

Reply:
Renamed!